pkg/db/db.go

```diff
@@ -0,0 +1,138 @@
+//Provides operations on SQLite DB
+package db
+
+import (
+	"database/sql"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+//Database type is sql.DB
+type Database sql.DB
+
+//Record type represents DB record description
+type Record struct {
+	ID, Name, Host, dataType, Data string
+}
+
+func (db *Database) sql() *sql.DB {
+	return (*sql.DB)(db)
+}
+
+//Connect connects to DB with given name database name
+func Connect(db_name string) (*Database, error) {
+	conn, err := sql.Open("sqlite3", db_name)
+	if err != nil {
+		return nil, err
+	}
+	db := (*Database)(conn)
+	if err := db.createTable(); err != nil {
```

Comment:
I think this is not the best idea, if you want to do something like this, it's better to check on start.